src/main/java/it/gov/pagopa/merchant/dto/InitiativeDTO.java

```diff
@@ -0,0 +1,26 @@
+package it.gov.pagopa.merchant.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class InitiativeDTO {
```

Comment:
add @NotNull/@NotEmpty annotation where supposed to be (when returning a DTO to a FE, because they use codeGen, the presence of such annotation help them in the typization)